src/config/proxy.ts

```diff
@@ -1,15 +1,27 @@
 import { envVars }  from "./env";
-import { bootstrap } from "global-agent";
-import { getLogger } from "./logger";
-
-const logger = getLogger("config.proxy");
-
-if (envVars.PROXY) {
-	logger.info(`configuring proxy: ${envVars.PROXY} for outbound calls`);
-	process.env.GLOBAL_AGENT_HTTP_PROXY = envVars.PROXY;
-	bootstrap();
-} else {
-	logger.info("configuring no proxy for outbound calls");
-}
```

Comment:
This was the global proxy config that would be used for ALL http calls. We had to do this globally, because we couldn't teach Probot to use a specific proxy. Now that Probot is replaced with our own GitHub clients, we can use a more targeted proxy config.